docs/sphinx/examples/python/bernstein_vazirani.py

```diff
@@ -1,113 +1,81 @@
-import argparse
 import cudaq
 import random
 
+from typing import List
+
 
 def random_bitstring(length: int):
-    bitstring = ""
-    for bit in range(length):
-        bitstring += str(random.randint(0, 1))
+    bitstring = []
+    for _ in range(length):
+        bitstring.append(random.randint(0, 1))
     return bitstring
```

Comment:
I agree that it's messy to call this list a bitstring. I didn't do the original translation (@khalatepradnya or @amccaskey  ?), but am I correct in thinking the type was changed because we don't support strings as kernel arguments? My issue is that Bernstein Vazirani deals with finding hidden strings; so "bitstring" is the most technically precise name from a physics POV

Reply:
Adding some points.

- bitstring is also a computer science term
   - that term implies a significance to the order, which a list does not; not clear if that is significant here, but we've had issues before
- string is a computer science term and within many contexts is indicative of the data structure (Python being one of those contexts)
- we should be careful not to mislead or confuse users with terminology

To be clear, I don't have any strong objection. Just want it to be clear as this is an example.

Reply:
I completely agree with your points. What are thoughts on: `hidden_bits`, `bit_list`, `bits` ? I can't think of anything better that satisfies both CS/Physics, but happy for suggestions 😃

Reply:
either one sounds good to me